scripts/mas_sso_config.sh

```diff
@@ -1,12 +1,10 @@
 #!/bin/bash
 
-KEYCLOAK_URL=http://localhost:8180
 TOKEN_PATH="/auth/realms/master/protocol/openid-connect/token"
-REALM=rhoas
 
 CLIENT_ID=admin-cli
-USERNAME=admin
-PASS=admin
+
+URL=$KEYCLOAK_URL:$KEYCLOAK_PORT_NO
```

Comment:
I'd suggest we have a variable that takes in the complete URL and not the host and port.
The advantage of having that is, for a deployed instance more often you do not need the port, just the host is enough. 
```suggestion
URL=$KEYCLOAK_URL
```

where KEYCLOAK_URL is a complete url, with host and port e.g `http://localhost:8180`

Reply:
the port number is also used in the `scripts/mas_sso_setup.sh` script. it might become inconsistent, when we provide full url

Reply:
@machi1990 ^^

Reply:
> the port number is also used in the `scripts/mas_sso_setup.sh` script. it might become inconsistent, when we provide full url

Yes, I think that's only where the port is used and I do not think the url is needed there.
It is okay to have not the same number of variables since it is for different make targets. 

I'll keep the port  to configure `sso/setup` target and `url` to configure the `sso/config` target.
The rest of the variables, username/password can be shared between the two targets.

Reply:
> > the port number is also used in the `scripts/mas_sso_setup.sh` script. it might become inconsistent, when we provide full url
> 
> Yes, I think that's only where the port is used and I do not think the url is needed there. It is okay to have not the same number of variables since it is for different make targets.
> 
> I'll keep the port to configure `sso/setup` target and `url` to configure the `sso/config` target. The rest of the variables, username/password can be shared between the two targets.

Thank you Manyanda, changing it in the sso/config! @machi1990 @pawelpaszki